NOTICE

```diff
@@ -78,7 +81,7 @@ This project includes:
   SLF4J API Module under MIT License
   StAX API under The Apache Software License, Version 2.0
   swe-common-data-framework under Mozilla Public License version 1.1
-  vecmath under GNU General Public License (GPL), version 2, with the CLASSPATH exception
+  vecmath under Sun Binary Code License Agreement
```

Comment:
Why does this change? There is no update of the dependencies, hence there should be no update of the licenses.